backend/tests/test_database.py

```
@@ -377,6 +377,43 @@ def test_preference_with_special_characters(self):
         self.assertEqual(value, special_value)
 
 
+class TestGetFilesByFaissIndices(unittest.TestCase):
+    """Tests for get_files_by_faiss_indices batch lookup function."""
+
+    def test_empty_list_returns_empty_dict(self):
+        """Empty input returns an empty dict without touching the DB."""
+        from backend import database
+        result = database.get_files_by_faiss_indices([])
+        self.assertEqual(result, {})
```

Ruff (0.15.1)

[warning] Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)